webpack.config.js

```diff
@@ -40,7 +40,6 @@ module.exports = {
         }
     },
     devServer: {
-        https: false,
```

Comment:
Is this required?

Reply:
@josejulio yeah, with the new version `https` is replaced with `server: "https"`, but in this case we can just omit it

The `start` script throws an error otherwise